### DISCUSSION TITLE
export textasset

### DISCUSSION THREAD

**Discussion by UNTAAA** [nesrak1/UABEA#399] [Q&A]
how to fix it?... I have exported but the text is like this

![Screenshot (138)](https://github.com/user-attachments/assets/2ac22d6b-0b04-4f00-9bb9-4d1a9654abd8)

**Comment by nesrak1** [comment]
This is a custom (non-unity engine) binary format. Looks like UABEA exported it fine. TextAssets can be binary files instead of text, which is what they did here.
